README.md

```diff
@@ -274,6 +274,13 @@ Server will then be accessible at http://localhost:8080/ and eg. http://localhos
           refuse_to_fetch_third_party_urls: false
           fhir_version: R4
 ```
+Also with custom property file and enviornment variables , use below docker command, replace <values> with actual values
```

Comment:
Suggest the following wording:
"You can use a custom property file that utilizes environment variables for many configuration properties. For example, to use the `application-custom.yaml` file. Replace <values> with actual values."